src/Data/Text/Internal/PrimCompat.hs

```diff
@@ -10,16 +10,23 @@ module Data.Text.Internal.PrimCompat
 
   , wordToWord32#
   , word32ToWord#
+
+  , isByteArrayPinned#
   ) where
 
 #if MIN_VERSION_base(4,16,0)
-
-import GHC.Base
-
+import GHC.Exts (wordToWord8#,word8ToWord#,wordToWord16#,word16ToWord#,wordToWord32#,word32ToWord#)
 #else
-
 import GHC.Prim (Word#)
+#endif
 
+#if MIN_VERSION_base(4,10,0)
```

Comment:
I think it's time for `base-4.9` (and corresponding GHC 8.0) to go: I'd rather drop its support than accumulate more boilerplate. @Lysxia what do you think?

Reply:
I also think we can drop base-4.9.

Reply:
I’m fine with killing off GHC 8.0 support. Should I open a separate PR doing that? I think it would be best for that to be a separate commit and not tied up in this addition.

Reply:
Yes, please make a separate PR to decomission GHC 8.0.

Reply:
Done.